### ISSUE TITLE
Update Cactoos version

### ISSUE THREAD

**Issue by carlosmiranda** [zerocracy/farm#1223]
Current Cactoos version of this project is [0.29](https://github.com/yegor256/cactoos/tree/0.29). This release is nearly six months old. Let's update to the latest version [0.35](https://github.com/yegor256/cactoos/tree/0.35), and make the appropriate code changes if necessary due to API change.

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by g4s8** [comment]
@carlosmiranda right, it looks like we have to change some imports and class names to be able to build the project.

**Comment by 0crat** [comment]
Job [#1223](https://github.com/zerocracy/farm/issues/1223) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)
